startup.d/10-git-pull.sh

```diff
@@ -1,2 +1,21 @@
 #!/bin/sh
+
+# Prepare code auditing environment
+REPO_URL=$(git remote get-url origin)
+TEST_DIR=$(mktemp -d)
```

Comment:
This should either be cleaned up so we're not wasting disk space or (preferably) use a single, fixed directory. With the latter option, you could fetch and hard reset to `origin/master` instead of cloning the entire repo every time.